src/java/org/apache/nutch/crawl/LinkDbMerger.java

```diff
@@ -68,12 +66,11 @@
  * 
  * @author Andrzej Bialecki
  */
-public class LinkDbMerger extends Configured implements Tool,
-    Reducer<Text, Inlinks, Text, Inlinks> {
+public class LinkDbMerger extends Configured implements Tool {
   private static final Logger LOG = LoggerFactory
       .getLogger(MethodHandles.lookup().lookupClass());
 
-  private int maxInlinks;
+  private static int maxInlinks;
```

Comment:
Similar to the situation in Generator: maybe it's clearer and safer to move the static variable to LinkDbMergeReducer as instance variable.